content/licenses/contributor-agreements.md

```diff
@@ -19,7 +19,12 @@ their work.
 * ICLA: [Individual Contributor License Agreement](icla.pdf)
 * CCLA: [Corporate Contributor License Agreement](cla-corporate.pdf)
 
-All contributors of ideas, code, or documentation to
+Contributions that are not significant may be accepted via Section 5
+of the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0)
+which states that any contribution intentionally submitted for inclusion
+shall be under the Apache License 2.0 unless otherwise specified.
+
+All contributors of significant ideas, code, or documentation to
```

Comment:
Not sure if there's a better adjective than "significant" -- perhaps we can pick something out of the formal definition of a de minimis-test https://www.lsd.law/define/de-minimis-test ?

Reply:
good change.

Reply:
A contribution is "significant" if reverting the change would be a significant amount of work for the project. Reverting the change might be needed if the contribution is found to be improper, such as occurs when the contributor is not actually the owner of the intellectual property, or the contributor owns a patent and later demands payment for royalties.